src/runtime/buffer_t.cpp

```diff
@@ -176,7 +176,7 @@ halide_buffer_t *_halide_buffer_crop(void *user_context,
 // with distinct store_at/compute_at levels. Each call to the stage
 // only fills in part of the buffer.
 HALIDE_BUFFER_HELPER_ATTRS
-void _halide_buffer_retire_crop_after_extern_stage(void *user_context,
+int _halide_buffer_retire_crop_after_extern_stage(void *user_context,
                                                    void *obj) {
```

Comment:
Unindent one space (or just put this on the same line, not worth a new line for ~10 chars).

another below